backend/app/api/ws/terminal_namespace.py

```
@@ -131,6 +132,33 @@ async def on_terminal_attach(self, sid: str, data: dict) -> dict:
             await self.leave_room(sid, _terminal_room(previous_session_id))
 
         await self.enter_room(sid, _terminal_room(session_id))
+        try:
+            attach_result = await get_sio().call(
+                "terminal:attach",
+                {"session_id": record.session_id},
+                to=record.socket_id,
+                namespace=DEVICE_NAMESPACE,
+                timeout=TERMINAL_ATTACH_TIMEOUT_SECONDS,
+            )
+        except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)